docs/mergeduty/howto.md

```diff
@@ -191,7 +221,7 @@ Make sure to double-check they match as that's generated automatically and somet
 
 1. [Close mozilla-beta](https://mozilla-releng.net/treestatus/show/mozilla-beta). Check _"Remember this change to undo later"_. Please enter a good message as the reason for the closure, such as "Mergeduty - closing beta for $VERSION RC week".
 1. Run the `m-b -> m-r` [no-op trial run](#do-migration-no-op-trial-runs) one more time, and show the diff to another person on releaseduty.
-1. The diff for `release` should be fairly similar to [this](https://hg.mozilla.org/releases/mozilla-release/rev/70e32e6bf15e), with updated branding as well as the version change.
+1. The diff for `release` should be fairly similar to [this](https://hg.mozilla.org/releases/mozilla-release/rev/0eae18af659f087056bce0f62a325e5e595fff72), with updated the version change.
```

Comment:
Thanks for cleaning up the branding part!

Nit: I'm confused `confvars.sh` doesn't change `ACCEPTED_MAR_CHANNEL_IDS`. Is this expected nowadays?

Reply:
I think so. It looks like @tomprince moved that logic into taskgraph: https://hg.mozilla.org/mozilla-central/rev/66f52bda7e14e26235bd0a43bb68ad11775046e4